src/search.cpp

```diff
@@ -926,7 +926,7 @@ namespace {
 
           // Futility pruning: parent node
           if (   predictedDepth < 7 * ONE_PLY
-              && ss->staticEval + futility_margin(predictedDepth) + 256 <= alpha)
+              && ss->staticEval + 256 + 200 * predictedDepth / ONE_PLY <= alpha)
```

Comment:
Before this line didn't depend on the value of ONE_PLY.  Wouldn't
```&& ss->staticEval + Value(200 * predictedDepth) + 256 <= alpha)```
be more consistent?